descriptors/ModuleDescriptor-template.json

```diff
@@ -29,6 +29,14 @@
     {
       "id": "policies",
       "version": "1.1"
+    },
+    {
+      "id": "role-capability-sets",
+      "version": "1.1"
+    },
+    {
+      "id": "role-capabilities",
+      "version": "1.1"
```

Comment:
only 1.0 is available https://github.com/folio-org/mod-roles-keycloak/blob/master/descriptors/ModuleDescriptor-template.json#L175